CONTRIBUTING.md

```diff
@@ -10,7 +10,7 @@ More details on all of these points may be found in the sections below.
 - [Sign-off](#license-and-copyright): all commits must be signed off.
 - [Tests](#tests): please ensure the PR includes adequate tests.
 - [Changelog](#documentation): if needed, please provide a changelog entry.
-- [Backports](#long-term-support-branches): provide a backport if needed (it's fine to wait until the main PR is accepted).
+- [Backports](#long-term-support-branches): Priority is given to the LTS branch over the development branch. After the PR has been integrated into the LTS branch, please merge it into the development branch.
```

Comment:
Err, not really.

`development` is where all development happens. Some PRs will be backported into the LTS branches, but not all. So all PRs must first start on `development`; it is not true that priority is given to LTS branches. And each PR targets only one branch (which is different to some other projects, such as OpenSSL, where a single PR is often merged to multiple applicable branches).

What is true is that we won't merge a PR to `development` if we want backport PRs and they are not present. Very specifically, gatekeepers (should) merge all PRs at the same time.

Now, some people will create the backports at the same time as the main PR. This means extra work if there are changes required.

The original phrase `it's fine to wait until the main PR is accepted` is correct. It does not say `merged`.

It is fine to wait until we are happy with a PR before creating backports. But it's usually best to start with a PR against `development` (obviously there are special cases of issues that are only in an LTS branch, but ignoring that for the sake of a generally-applicable rule) and then create backport PRs if required/requested

Reply:
@tom-cosgrove-arm Thank you for clarifying a lot more about the workflow. Your comments are very good, and I think they should be in `CONTRIBUTING.md`. I have submitted a new commit, adding most of your comments.

Sorry for the forced-commit. The PR is totally different from the original change, and I thought the original commit was not necessary to keep it.